### PR TITLE
Update to v0.0.4

### PR DESCRIPTION
**Pull request by marcoesters**
# Changes

* Update to v0.0.4
* Make requirements consistent with upstream
* Update tests
* Change recipe format to comply with new recipe standards

# Package Info

Upstream: https://github.com/anaconda-distribution/anaconda-linter/tree/0.0.4
Change log: https://github.com/anaconda-distribution/anaconda-linter/blob/0.0.4/CHANGELOG.md

### CODE REVIEW

**Review by ous8292 — APPROVED** [review]
LGTM

File: recipe/meta.yaml [hunk 1]
```diff
@@ -21,37 +21,42 @@ requirements:
...
 test:
   source_files:
     - tests
   requires:
+    - pip
     - pytest
-    - pytest-cov
-    - pytest-html
+  imports:
+    - anaconda_linter
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:42]: Could you add  `- anaconda_linter.lint` too?

Comment (marcoesters) [reply recipe/meta.yaml:42]: Added, thank you!

**Review by skupr-anaconda — APPROVED** [review]
LGTM

**Pull request merged**